internal/kibana/spaces/schema.go

```diff
@@ -51,6 +51,10 @@ func (d *dataSource) Schema(_ context.Context, _ datasource.SchemaRequest, resp
 							Description: "The data-URL encoded image to display in the space avatar.",
 							Optional:    true,
 						},
+						"solution": schema.StringAttribute{
```

Comment:
This field is only supported from Elastic Stack 8.18.0 and higher. You'll need to check that the provider is running against a supported server version. You can check `internal/kibana/slo.go` line 701-709 for an example of how to do this.